app/controllers/info_controller.rb

```diff
@@ -11,23 +11,22 @@ class InfoController < ApplicationController
   def show
     @slug = URI.encode(params[:slug])
 
-    begin
-      metadata = GOVUK::Client::MetadataAPI.new.info(@slug).to_h
-    rescue StandardError
-      metadata = nil
-    end
-
     @content = content_store.content_item("/#{@slug}").to_h
     @needs = @content["links"]["meets_user_needs"]
 
-    if metadata
+    begin
+      metadata = GOVUK::Client::MetadataAPI.new.info(@slug).to_h
+
       document_type = @content["document_type"] || @content["format"]
 
       part_urls = get_part_urls(@content, @slug)
       @is_multipart = is_multipart(part_urls, document_type)
       calculated_metrics = metrics_from(metadata.fetch("performance"), part_urls, @is_multipart)
       @lead_metrics = calculated_metrics[:lead_metrics]
       @per_page_metrics = calculated_metrics[:per_page_metrics]
+    rescue StandardError => e
+      logger.error "Error fetching metadata for #{@slug} from the Metadata API"
```

Comment:
my only pedantic comment is that the error message is perhaps a bit too specific - the fetching of the metadata is really only happening in line 18. As far as I can tell, the intent of this action is to ultimately consume the metadata and create metrics. Perhaps an error message to that effect would be more descriptive of what could possibly have gone wrong?